packages/field-content/src/views/editor/index.js

```diff
@@ -44,12 +44,12 @@ function Stories({ value: editorState, onChange, blocks, className }) {
           if (typeof block.getPlugins !== 'function') {
             return combinedPlugins;
           }
-          return [...combinedPlugins, block.getPlugins({ blocks })];
+          return [...combinedPlugins, ...block.getPlugins({ blocks })];
```

Comment:
Fixing a bug introduced in #1358 where we weren't correctly flattening the plugins returned.